packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -2893,10 +2945,13 @@ class _MenuItemLabel extends StatelessWidget {
           ),
       ],
     );
-    if (semanticsLabel != null) {
+    if (semanticsLabel != null || isEnabled != null || isSemanticButton != null) {
       menuItemLabel = Semantics(
         label: semanticsLabel,
-        excludeSemantics: true,
+        enabled: isEnabled,
+        button: isSemanticButton,
```

Comment:
I'm not sure that this is the right place to set the semantics `button` property. I think we should be replacing the `isSemanticButton: null` passed to `TextButton` above (line 1886).

Same for `enabled`.